services/harmony/test/parameters/averaging_type.ts

```diff
@@ -0,0 +1,107 @@
+import { expect } from 'chai';
+// import { hookRedirect } from '../helpers/hooks';
+import { hookRangesetRequest } from '../helpers/ogc-api-coverages';
+import hookServersStartStop from '../helpers/servers';
+import StubService from '../helpers/stub-service';
+
+// Note that there are currently no time or area averaging services in services.yml
+// which is why several of the assertions are using `xit`. Once there are services to
+// support it we should enable the assertions and uncomment out the code to follow
+// the redirects.
+describe('averagingType', function () {
+  const collection = 'C1233800302-EEDTEST';
+  hookServersStartStop();
+
+  describe('when making a request with averagingType time', function () {
+    const averagingTimeQuery = {
+      averagingType: 'time',
+    };
+
+    describe('for a collection that can support it', function () {
+      StubService.hook({ params: { redirect: 'http://example.com' } });
+      hookRangesetRequest('1.0.0', collection, 'all', { query: { ...averagingTimeQuery } });
+      // hookRedirect('anonymous');
```

Comment:
nit: remove commented out code

Reply:
We will uncomment these lines once there are time and area averaging services and switch the `xit` to `it`. I put a comment up above the start of the tests - maybe I should also add another comment directly above each commented out line.

Note the reason they are commented out is because they will break unless the request gets a 3xx redirect back which won't happen until the request gets matched to a service.